data/korath/korath variants.txt

```diff
@@ -791,8 +791,8 @@ ship "Ra'gru Ik 618" "Ra'gru Ik 618 (Sestor)"
 	gun "Korath Detainer"
 	gun "Korath Detainer"
 	gun "Korath Detainer"
```

Comment:
Should probably have a variant that goes all piercers... maybe.  I delegated a lot of the variant/outfitting

```suggestion
	gun "Korath Detainer"
	pylon
	gun "Korath Detainer"
	pylon
	gun "Korath Detainer"
	pylon
```